build.sbt

```diff
@@ -1,65 +1,88 @@
+    "-unchecked",                    // warn about unchecked type parameters
+    "-feature",                      // warn about misused language features
+    "-language:higherKinds",         // allow higher kinded types without `import scala.language.higherKinds`
     "-language:implicitConversions", // allow use of implicit conversions
     "-language:postfixOps",
-    "-Xlint", // enable handy linter warnings
-    "-Xfatal-warnings", // turn compiler warnings into errors
-    "-Ypartial-unification", // allow the compiler to unify type constructors of different arities
+    "-Xlint",             // enable handy linter warnings
+    "-Xfatal-warnings",   // turn compiler warnings into errors
     "-Ywarn-macros:after" // allows the compiler to resolve implicit imports being flagged as unused
   )
 )
 
-lazy val artifactorySettings = Seq(
-  resolvers += "Artifactory" at "https://moda.jfrog.io/moda/sbt",
-  credentials += {
-    (sys.env.get("JFROG_USER"), sys.env.get("JFROG_PASS")) match {
-      case (Some(user), Some(pass)) => Credentials("Artifactory Realm", "moda.jfrog.io", user, pass)
-      case _ => Credentials(Path.userHome / "credentials.sbt")
-    }
-  }
-)
+lazy val root = (project in file("."))
+  .aggregate(`state-machine`, examples)
+  .settings(
+    publishArtifact := false
+  )
 
 lazy val `state-machine` =
   (project in file("state-machine"))
     .settings(
       commonSettings,
-      artifactorySettings,
       Dependencies.TestLib,
       Dependencies.Logging,
       addCommandAlias("format", ";scalafmt;test:scalafmt;scalafmtSbt")
     )
-    .enablePlugins(GitVersioning)
 
 lazy val examples =
   (project in file("examples"))
     .settings(
       commonSettings,
-      artifactorySettings,
       Dependencies.TestLib,
       Dependencies.Logging,
       Dependencies.FS2,
       addCommandAlias("format", ";scalafmt;test:scalafmt;scalafmtSbt")
     )
-    .enablePlugins(GitVersioning)
+    .settings(
+      publishArtifact := false
+    )
     .dependsOn(`state-machine`)
+
+inThisBuild(
+  List(
+    licenses := Seq(
+      "MIT" -> url("https://raw.githubusercontent.com/laserdisc-io/fs2-aws/master/LICENSE")
+    ),
+    homepage := Some(url("https://github.com/modaoperandi/sc-state-machine/")),
+    developers := List(
+      Developer("semenodm", "Dmytro Semenov", "", url("https://github.com/semenodm"))
+    ),
+    scmInfo := Some(
+      ScmInfo(
+        url("https://github.com/modaoperandi/sc-state-machine/tree/master"),
+        "git@github.com:ModaOperandi/sc-state-machine.git",
+        "git@github.com:ModaOperandi/sc-state-machine.git"
+      )
+    ),
+    publishMavenStyle               := true,
+    Test / publishArtifact          := true,
+    pomIncludeRepository            := (_ => false),
+    pgpPublicRing                   := file(".circleci/local.pubring.asc"),
+    pgpSecretRing                   := file(".circleci/local.secring.asc"),
```

Comment:
will these exist on circle?

Reply:
no, that's the problem with circle, that's why o release it from local machine now